### PR TITLE
Changelog for edge-19.2.5 release

### PR DESCRIPTION
**Pull request by ihcsim**
Signed-off-by: Ivan Sim <ivan@buoyant.io>

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,3 +1,28 @@
+## edge-19-2.5
+
+* CLI
+  * Updated `linkerd check` to ensure hint URLs are displayed for RPC checks
+* Controller
+  * Updated the auto-inject admission controller webhook to respond to UPDATE
+    events of deployment workload
```
Comment (klingerf) [inline CHANGES.md]: Tiny nit -- how about rewording this as "events for deployment workloads"?

File: CHANGES.md [hunk 2]
```diff
@@ -1,3 +1,28 @@
...
+  * Updated destination service to return TLS identities only when the client
+    pod is TLS-aware and is in the same controller namespace
```
Comment (klingerf) [inline CHANGES.md]: Rather than calling this the "client pod", I think it makes more sense to call it the "destination pod".

File: CHANGES.md [hunk 3]
```diff
@@ -1,3 +1,28 @@
...
+  * Lessen klog level to improve security
+  * Updated control-plane components to query Kubernetes at startup to determine
+    authorized namespaces and if ServiceProfile support is available
+  * Modified the stats payload to include the following TCP stats:
+    `tcp_open_connections`, `tcp_read_bytes_total`, `tcp_write_bytes_total`
+* Proxy
+  * Fixed issue with proxy falling back to filesystem polling due to improperly
+    sized inotify buffer
+* Web UI
```
Comment (klingerf) [inline CHANGES.md]: Let's add one more bullet point in this section for the TCP stats chance, since we're now displaying TCP stats in the web UI, and it's great. Something like:

```
* Updated the resource detail page to start displaying a table with TCP stats
```

File: CHANGES.md [hunk 4]
```diff
@@ -1,3 +1,29 @@
...
+  * Removed 'Help' hierarchy and surfaced links on navigation sidebar
+  * Added an Endpoints view to the web dashboard
```
Comment (rmars) [inline CHANGES.md]: How about `Added a Debug page to the web dashboard, allowing you to introspect service discovery state`?
